dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -0,0 +1,57 @@
+/* global $ skipSound preTitle lastServerResponse Dialog */
```

Comment:
I like that it becomes more explicit what our global are (so that we can know what we need to improve) :)